src/Browsershot.php

```diff
@@ -254,6 +257,26 @@ public function setProxyServer(string $proxyServer)
         return $this;
     }
 
+    public static function jsLinksInWhitelist(string $html, $safeDomains = []) {
+        // https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#php
+        libxml_set_external_entity_loader(null);
```

Comment:
Will this have side effect for people that use `DomDocument` elsewhere in the same script?

Reply:
What do you mean? I can't understand this part.

Reply:
Image that this code is run by someone who set this to a specific option. After running our code, that option isn't set anymore. Should we restore it to the value it was previously?